scip.proto

```diff
@@ -327,6 +327,17 @@ message SymbolInformation {
   // while other fields such as `Documentation.occurrences` can be optionally
   // included to support hyperlinking referenced symbols in the signature.
   Document signature_documentation = 7;
+  // (optional) The enclosing symbol if this is a local symbol.  For non-local
+  // symbols, the enclosing symbol should be parsed from the `symbol` field
+  // using the `Descriptor` grammar.
+  //
+  // The primary use-case for this field is to allow local symbol to be displayed
+  // in a symbol hierarchy for API documentation. It's OK to leave this field
+  // empty for local variables since local variables usually don't belong in API
+  // documentation. However, in the situation that you wish to include a local
+  // symbol in the hierarchy, then you can use `enclosing_symbol` to locate the
+  // "parent" or "owner" of this local symbol.
+  string enclosing_symbol = 8;
```

Comment:
I debated using `owner` or `parent`, which is commonly the name of this functionality in compiler APIs that I have worked with (Java, Scala, C#, and TypeScript). However, I went with `enclosing_symbol` to make it more self-descriptive and also to reflect that I don't expect this to be a high-traffic API (unlike `owner` in the compiler that you use all the time).